superset/assets/javascripts/dashboard/reducers/index.js

```diff
@@ -0,0 +1,29 @@
+import { combineReducers } from 'redux';
+import shortid from 'shortid';
+
+import charts from '../../chart/chartReducer';
+import dashboardState from './dashboard';
+import datasources from './datasources';
+import sliceEntities from './sliceEntities';
+import dashboardLayout from '../v2/reducers/index';
```

Comment:
should you update the `/v2/reducers/index` file to be called `dashboardLayout`?

Reply:
`dashboardLayout` is basic layout reducer. `index.js` is wrapping `dashboardLayout` with `undoable` HOC.